runtime/flang/fmtconv.c

```diff
@@ -123,7 +131,7 @@ __fortio_default_convert(char *item, int type,
   case __REAL4:
     width = REAL4_W;
     (void) __fortio_fmt_g((__BIGREAL_T)PP_REAL4(item), width, REAL4_D, REAL4_E,
```

Comment:
A question about consistency: in the `assign.c` file all uses of __BIGREAL_T were changed to __REAL16_T. Shouldn't it be done here too?

Reply:
I have reverted the `__REAL16_T` changes, so that the runtime code continues to use `__BIGREAL_T` (reducing the delta). `__BIGREAL_T` is still 64-bit on targets that don't support QuadFP, but 128-bit on targets that do.